packages/terra-clinical-result/src/flowsheetresultcell/FlowsheetResultCell.jsx

```diff
@@ -0,0 +1,370 @@
+    diastolic: !isEmpty(diastolic) ? checkIsStatusInError(diastolic.status) : false,
+  };
+  if (isStatusInError.systolic || isStatusInError.diastolic) {
+    resultsInnerDisplay = <EnteredInError />;
+  } else {
+    resultsInnerDisplay = (<ClinicalResultBloodPressure key={(`ClinicalResultBloodPressure-${resultKeyID}`)} resultData={resultDataItem} hideUnit={hideUnit} isTruncated />);
+  }
+  const clinicalResultDisplay = createClinicalResultDisplay(resultsInnerDisplay, hasUnverifiedIcon, hasInterpretationIcon, containerDivRef, resultKeyID);
+  return clinicalResultDisplay;
+};
+
+const setResultKeyID = (isBloodPressureResult, resultData) => {
+  if (isBloodPressureResult) {
+    if (resultData.id) {
+      return resultData.id;
+    }
+    if (!isEmpty(resultData.systolic) && resultData.systolic.eventId) {
+      return resultData.systolic.eventId;
+    }
+    if (!isEmpty(resultData.diastolic) && resultData.diastolic.eventId) {
+      return resultData.diastolic.eventId;
+    }
+  } else {
+    if (resultData.eventId) {
+      return resultData.eventId;
+    }
+    if (resultData.id) {
+      return resultData.id;
+    }
+  }
+  return null;
+};
+
+const AttributesTemplate = (interpretationValue = false, commentBool = false, modifiedBool = false, unverifiedBool = false) => ({
+  interpretation: !!(interpretationValue),
+  comment: commentBool,
+  modified: modifiedBool,
+  unverified: unverifiedBool,
+});
+
+const unpackResultAttributes = (resultDataItem) => {
+  const {
+    interpretation,
+    hasComment,
+    isModified,
+    isUnverified,
+  } = resultDataItem;
+  const itemAttributes = new AttributesTemplate();
+  itemAttributes.interpretation = interpretation;
+  itemAttributes.comment = hasComment;
+  itemAttributes.modified = isModified;
+  itemAttributes.unverified = isUnverified;
+  return itemAttributes;
+};
+
+const unpackResultDataSet = (resultDataSet) => {
+  const systolicData = resultDataSet[0].systolic;
+  const diastolicData = resultDataSet[0].diastolic;
+  const isfirstBloodPressureResult = (!isEmpty(diastolicData) || !isEmpty(systolicData)) || false;
+  let firstResultAttributes = {};
+  let firstResultData = {};
+  if (isfirstBloodPressureResult) {
+    const bpAttribute = {
+      systolic: null,
+      diastolic: null,
+    };
+    bpAttribute.systolic = !isEmpty(systolicData) ? unpackResultAttributes(systolicData) : new AttributesTemplate();
+    bpAttribute.diastolic = !isEmpty(diastolicData) ? unpackResultAttributes(diastolicData) : new AttributesTemplate();
+    firstResultAttributes = new AttributesTemplate(
+      (bpAttribute.systolic.interpretation),
+      (bpAttribute.systolic.comment || bpAttribute.diastolic.comment),
+      (bpAttribute.systolic.modified || bpAttribute.diastolic.modified),
+      (bpAttribute.systolic.unverified || bpAttribute.diastolic.unverified),
+    );
+    firstResultData = JSON.parse(JSON.stringify(resultDataSet[0]));
```

Comment:
Here and below (252), these JSON.parse(JSON.stringify()) lines break our consumption of this component.  We are handling localication and formatting of numeric values and handing in a node to be placed in.  This results in an error being thrown during stringify() when attempting to handle that value.

What is terra's stance on numeric value localization (e.g. number: 1000.123 en-US: 1,000.123 de-DE: 1.000,123, sv-SE: 1 000,123) and what is the expectation for consumers around this?  Is this something where consumers should supply a pristine string value received from a source of truth to avoid IEEE-754 hiccups along the way and allow this UI to apply localization?

What do you see as the go forward for consumers to adhere to OCS in these scenarios?